### ISSUE TITLE
AttributeError: 'NexUpload' object has no attribute '_logger'

### ISSUE THREAD

**Issue by waza-ari** [brainelectronics/micropython-nextion#31]
I guess related to commit 9406898f0cc04a7541112879aba8e8d99fc5dfc3, running the upload example throws the following exception.

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "upload.py", line 30, in <module>
  File "/lib/nextion/nextion_upload.py", line 42, in __init__
  File "/lib/nextion/nextion_upload.py", line 130, in _checkFile
AttributeError: 'NexUpload' object has no attribute '_logger'
```

Looking at the source code, I don't understand how the changes in 9406898f0cc04a7541112879aba8e8d99fc5dfc3 are supposed to work, so I'm not sure I'll be able to provide a PR for this, as I don't understand what the intentions are :(

**Comment by waza-ari** [comment]
I can confirm reverting the changes from https://github.com/brainelectronics/micropython-nextion/commit/9406898f0cc04a7541112879aba8e8d99fc5dfc3 does fix the issue.

**Comment by brainelectronics** [comment]
WIP, fix is on the way

**Comment by brainelectronics** [comment]
Fixed with [0.15.2](https://github.com/brainelectronics/micropython-nextion/releases/tag/0.15.2)
